Add initial bootstrap governance

afrittoli opened the pull request:
Add an initial governance.md to define the bootstrap governance
committee

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

afrittoli commented on governance.md:
```diff
@@ -0,0 +1,97 @@
+
+Members are (in alphabetical order):
+
+* Emil Bäckmark, [@e-backmark-ericsson](https://github.com/e-backmark-ericsson), Ericsson
+* Andrea Frittoli, [@afrittoli](https://github.com/afrittoli), IBM
+* Mattias Linnér, [@m-linner-ericsson](https://github.com/m-linner-ericsson), Ericsson
+* Erik Sternerson [@erkist](https://github.com/erkist), doWhile
+* Steve Taylor [@sbtaylor15](https://github.com/sbtaylor15), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
```
The list here is based on who more frequently (and recently) attended the sig-events vocabulary meeting.
Feel free to comment if you'd prefer not to be in this list or to be added to it - we can discuss about this in the next meeting too. We don't necessarily need to have 5 individuals either - this is what the Tekton project did and it worked well there.

afrittoli commented on governance.md:
These are two examples here, but we will end-up documenting our communication channels in the main README (and/or in community.md)

oleg-nenashev reviewed (changes requested):
The title should be CD Events not Tekton

As discussed at the CD Events SIG, it might make sense to have this file in .github so that it can be referenced from all repositories.

m-linner-ericsson commented on governance.md:
```diff
@@ -0,0 +1,97 @@
+  * Who is authorized to release and when.
+  * Frequency limits
+* Defines the documentation process
+* Defines what CLA process is required and how it is programmatically enforced before code is merged
```
I think we should spell out abbreviations such as CLA or link to some page describing it e.g. https://en.wikipedia.org/wiki/Contributor_License_Agreement

afrittoli replied:
done

m-linner-ericsson commented on governance.md:
```diff
@@ -0,0 +1,97 @@
+  * Frequency limits
+* Defines the documentation process
+* Defines what CLA process is required and how it is programmatically enforced before code is merged
+
```
Should the bootstrap committee also set up a security process? What if someones finds a security issue, how is this reported and handled?

From the [Eiffel Bootstrap committee governance](https://github.com/eiffel-community/community/commit/63883403ebcf07af3b312af103a0579819d8409c#diff-b60c6a93e9f74ee52e71bf33781c2870b8395c8c5a34f00f61305343cb8d8447)

```suggestion
### Security/Vulnerability Reporting and Response Process
* Identify and document where vulnerability reporting can be done to the project
* Identify and document who is responsible for receiving vulnerability reports
* Document process responsible parties go through to triage and determine veracity of vulnerability
* Document process for facilitating fix, generating release update, and communicating vulnerability and fix to public
```

afrittoli replied:
Sure, I'm happy to add that

afrittoli replied:
done

m-linner-ericsson commented:
> As discussed at the CD Events SIG, it might make sense to have this file in .github so that it can be referenced from all repositories.

I don't know if it was @oleg-nenashev or @sbtaylor15 who mentioned this. 

When looking again at the `.github` repository I didn't find any good reason for putting the governance there. I know it's good for code of conduct and contributor guidelines as GitHub automatically links it, but I can't see any purpose with the governance. Am I missing something here?

oleg-nenashev commented:
It is basically style topic. As you prefer. I just do not want to have governance being version along with the spec tags. Sorry moving it to a separate repo makes sense to me

afrittoli commented:
> 



> It is basically style topic. As you prefer. I just do not want to have governance being version along with the spec tags. Sorry moving it to a separate repo makes sense to me

The governance docs are not really versioned, so I'm not sure I see the issue?

We could have a dedicated repo for community / governance type of documents, owned by the project governance team, but that seemed a bit of an overhead to just get started - I wanted to avoid spending too much time in managing a lot of repos :P 

I you feel strongly about using the `.github` repo for this, it works for me.

afrittoli commented on governance.md:
```diff
@@ -0,0 +1,97 @@
+# Tekton Bootstrap Governance
```
Heh, good catch, thank you!

afrittoli commented:
I addressed all comments. 
I also made a PR to the `.github` repo: https://github.com/cdevents/.github/pull/1
We can decide next time we meet which of the two to merge, or if we shall create a new repo to host this.

oleg-nenashev reviewed (approved):
Time to merge it? All the text looks reasonable to me. IMHO there are too many lists planned, maybe some of them could be moved to GitHub Issues or discussions. The Code of Conduct escalations handing could be also directly delegated to the CDF, but no strong opinion here

e-backmark-ericsson commented:
Merging this as a base for our govenance model
